src/ConfigTemplateMerge.php

```diff
@@ -18,6 +18,7 @@ class ConfigTemplateMerge {
   const ADD_VALUE = 1;
   const ADD_KEY = 2;
   const CHANGE_VALUE = 3;
+  const UPDATE_VALUE = 4;
```

Comment:
Same as with variables and functions it would be better to provide the public access explicitly.

Reply:
fixed in #37